src/main/java/core/basesyntax/db/DatabaseImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.db;
+
+import core.basesyntax.models.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class DatabaseImpl implements Database {
+    private static Map<Fruit, Integer> storage = new HashMap<>();
```

Comment:
```suggestion
public class StorageDaoImpl implements StorageDao {
```
краще мапу винести в окремий клас Storage